### ISSUE TITLE
ChatSecure -> ChatSecure, messages won't show up on receiver's end.

### ISSUE THREAD

**Issue by pks** [ChatSecure/ChatSecure-iOS#751]
When using OMEMO (deemed automatically as 'Best Available' encryption) to encrypt messages sent between two ChatSecure clients (both running on 4.0.7) messages fail to show up on the receiver's end. They're shown locally however. Plaintext and OTR work.

The server is from account.conversatons.im.

**Comment by chrisballinger** [comment]
I'm pretty sure you're seeing this issue. https://github.com/ChatSecure/ChatSecure-iOS/issues/743

**Comment by pks** [comment]
Thanks! How could I verify?

**Comment by chrisballinger** [comment]
There's no way to surface those errors in the UI unfortunately, but I'm regularly seeing the behavior you're describing in the debugger so it's only a matter of time until I figure out the root of the problem.

**Comment by pks** [comment]
I'm also not able to send any media (regardless of the client of chat partner) -- the icon that should be left of the input is just not there. Could that be related?

**Comment by chrisballinger** [comment]
That's something else, track the XEP-0363 feature issue

**Comment by chrisballinger** [comment]
This should be resolved in 4.0.8. Currently held up in review...

**Comment by nicnocquee** [comment]
I tested the last commit (cfb089df) on iPhone 6 plus and simulator and it works! Thanks!

**Comment by afriedmanGlacier** [comment]
Works for me too! Great job @chrisballinger!

**Comment by HummusSamurai** [comment]
Updated from the app store, and it seems to be working now.

On a different note, is it normal for received messages to show up on only 1 online device and not on both? Both can send and receive messages fine, but can't both get a message if they're both online.

Thank you for fixing this horrible bug! It's been awkward with the people that I've convinced to use ChatSecure and any longer and I'd have reverted to bad ol' Signal.
